### PR TITLE
feat(video): Minimise/maximise videos

### PR DESCRIPTION
**Pull request by ljkbennett**
https://github.com/ItsJuice/juicebox/projects/1#card-394326

### CODE REVIEW

File: apps/juicebox_web/assets/js/videos/video.scss [hunk 1]
```diff
@@ -1,13 +1,41 @@
...
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 50px;
+  height: 50px;
+  cursor: pointer;
+}
+
+.video-title {
+  width: 100%;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/videos/video.scss]: Properties should be ordered bottom, display, width, height, padding, font-size, background-color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/videos/video.scss [hunk 2]
```diff
@@ -1,13 +1,41 @@
...
   position: relative;
-  padding-bottom: 56.25%; /* 16:9 */
-  padding-top: 25px;
-  height: 0;
 }
+
+.video-expanded.video-present {
+  .video-wrapper {
+    position: relative;
+    padding-bottom: 56.25%; /* 16:9 */
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/videos/video.scss]: Use `//` comments everywhere

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#comment)

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/videos/video.scss]: Properties should be ordered position, height, padding-top, padding-bottom

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

**Comment by littleowllabs** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).
* 1 fixed issue! 🎉

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/71.

File: apps/juicebox_web/assets/js/videos/video.scss [hunk 3]
```diff
@@ -1,13 +1,41 @@
...
-  padding-top: 25px;
-  height: 0;
 }
+
+.video-expanded.video-present {
+  .video-wrapper {
+    position: relative;
+    height: 0;
+    padding-top: 25px;
+    padding-bottom: 56.25%; /* 16:9 */
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/videos/video.scss:11]: Use `//` comments everywhere

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#comment)

**Comment by ljkbennett** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 2 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/71.

**Review by OCannings — APPROVED** [review]

**Pull request merged**